pkg/version/version.go

```diff
@@ -51,12 +51,17 @@ func queryPackageVersion(cmdArg ...string) string {
 				if outp, err := cmd.Output(); err == nil {
 					output = string(outp)
 				}
+			} else if cmdArg[0] == "/usr/bin/pacman" {
```

Comment:
could you change this section to a switch statement?

Reply:
Sure, done.